packages/ui/src/proposals/modals/AddNewProposal/AddNewProposalWarningModal.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+import { ButtonPrimary } from '@/common/components/buttons'
+import { Checkbox } from '@/common/components/forms'
+import { Arrow } from '@/common/components/icons'
+import { AlertSymbol } from '@/common/components/icons/symbols'
+import { Modal, ModalBody, ModalFooter, ModalHeader } from '@/common/components/Modal'
+import { TextMedium } from '@/common/components/typography'
+import { BorderRad, Colors } from '@/common/constants'
+import { useModal } from '@/common/hooks/useModal'
+
+interface AddNewProposalWarningModalProps {
+  onNext: () => void
+}
+
+export const AddNewProposalWarningModal = ({ onNext }: AddNewProposalWarningModalProps) => {
+  const { hideModal } = useModal()
+  const [isAwareChecked, setAwareChecked] = useState(false)
+
+  return (
+    <Modal modalSize="m" modalHeight="s" onClose={hideModal}>
+      <ModalHeader onClick={hideModal} title="Caution" icon={<AlertSymbol />} />
+      <ModalBody>
+        <TextMedium margin="s">
+          Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim
+          velit mollit. Exercitation veniam consequat sunt nostrud amet.
+        </TextMedium>
+        <TextInfo>
```

Comment:
this should be `Info` component from `'@/common/components/Info'`

Reply:
I see also that the title is not needed here, so you can make it optional for that component as well.

Reply:
✅